.github/workflows/reviewdog-k8s.yml

```diff
@@ -47,6 +48,29 @@ jobs:
           unzip kube-linter-linux.zip
         env:
           KUBELINTER_VERSION: 0.1.2
-      - name: kubelinter
+      - name: kubelinter for k8s directory
         run: |
-          ./kube-linter lint --config .github/kubelinter.yaml k8s
+          ./kube-linter lint \
```

Comment:
how about define command in the makefile?

and it would be better to easily update kubelinter version.
latest version is v0.1.6

Reply:
added a task to Makefile.
also i added it to ci-container build. it will be available ci-container in nightly version once this PR is merged.

Reply:
thank you